test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
> So, unless there's a compelling reason to make the change to the test file (reducing the number of test cases in the array

Before this PR when the test case in question was an expected-fail it did fail most (all?) of the time because only around 64KB of data instead of the 138MB for just one step of the test.

When this test was converted to a passing test it was processing on the order of 500MB+ which I think is a bit excessive. It was making some test systems time out.

I understand what you're saying. I don't disagree with you. But there is some middle ground.